DependencyInjection/Configuration.php

```diff
@@ -388,6 +403,14 @@ private function addExceptionSection(ArrayNodeDefinition $rootNode)
                             ->useAttributeAsKey('name')
                             ->prototype('boolean')->end()
                         ->end()
+                        ->booleanNode('show_exception')
+                            ->treatNullLike($this->debug)
```

Comment:
Dont see much benefits in moving to extension:
```
$showException = (null === $config['exception']['show_exception']) ? $container->getParameter('kernel.debug') : $config['exception']['show_exception'];
$container->getDefinition('fos_rest.exception.controller')->replaceArgument(2, $showException);
```
Looks way more complicated.
Actually i used same approach as in symfony docs - http://symfony.com/doc/current/cookbook/configuration/using_parameters_in_dic.html